src/Uno.UI.RuntimeTests/Tests/Windows_Storage/Given__StorageFolder_GetItems.cs

```diff
@@ -0,0 +1,133 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.Resources;
+using Windows.ApplicationModel.Resources.Core;
+using System.Linq;
+
+namespace Uno.UI.RuntimeTests.Tests
+{
+	[TestClass]
+	public class Given__StorageFolder_GetItems

```

Comment:
```suggestion
	public class Given_StorageFolder_GetItems

```

Reply:
Of course, this was bad copy/paste operation :)